bitbucket-slack-server-integration-plugin/pom.xml

```diff
@@ -482,6 +481,12 @@
                     <jvmArgs>${jvm17.opens} -Djdbc.url=jdbc:h2:${project.build.directory}/bitbucket/home/shared/data/db;DB_CLOSE_DELAY=-1;AUTO_SERVER=TRUE</jvmArgs>
                     <jvmDebugPort>5007</jvmDebugPort>
 
+                    <banningExcludes>
+                        <exclude>com.atlassian.security:atlassian-secure-random</exclude>
+                        <exclude>com.atlassian.fugue:fugue</exclude>
+                        <exclude>com.google.code.gson:gson</exclude>
```

Comment:
Looks like this is the only way forward when upgrading amps - https://community.developer.atlassian.com/t/com-google-code-gson-gson-is-a-banned-dependency/78867